docs/CONTRIBUTING.md

```diff
@@ -84,31 +85,16 @@ To return to you home root environment::
 
 See the full conda docs [here](http://conda.pydata.org/docs).
 
-At this point you can easily do a *development* install, as detailed in the next sections.
-
-### 3. Installing Dependencies
-
-To run *r5py* in an development environment, you must first install
-*r5py*'s dependencies. We suggest doing so using the following commands
-(executed after your development environment has been activated)
-to ensure compatibility of all dependencies:
-
-    conda config --env --add channels conda-forge
-    conda config --env --set channel_priority strict
-    conda install configargparse geopandas joblib jpype1 pandas psutil requests pytest pytest-cov codecov requests black
-
-This should install all necessary dependencies including optional and packages for running tests.
-
-### 4. Making a development build
+### 3. Making a development build
 
 Once dependencies are in place, make an in-place build by navigating to the git
-clone of the *r5py* repository and running:
+clone of the *r5py* repository and run:
 
     pip install .
```

Comment:
This is fine for now, but there should be better ways (like adding `src/` to PYTHON_PATH, and running straight from source code). Let’s come back to this at some point.

Reply:
Yep, that is the other option, but this is propably easier approach for beginners. I guess we could mention both ways.